Simplified/Reader2/AdobeDRM/AdobeDRMLibraryService.swift

```diff
@@ -25,15 +25,15 @@ class AdobeDRMLibraryService: DRMLibraryService {
   /// - Parameter file: file URL
   /// - Returns: Always returns true
   func canFulfill(_ file: URL) -> Bool {
-    return true
+    return file.path.hasSuffix("_rights.xml")
```

Comment:
ADEPT library is always looking for a license file with the same file name and "_rights.xml" suffix for license info. This suffix is defined in adept_filter.cpp file, if this file exists, it means we downloaded a book protected with Adobe DRM. `return true` was used to make Readium 2 streamer part work with our Adobe DRM library. Now we can open LCP and Adobe DRM protected files, that part should not return true.

Also I see I need to update documentation to the method, as it says Always return true

Reply:
sounds good, if it's already defined maybe we can expose it and refer to it from other places? Since `adept_filter.cpp ` is only linked by SimplyE, I think in another PR I was suggesting to also define it in r2-shared perhaps? I don't really care where, but as long as it not hardcoded everywhere it would be an improvement.

Reply:
Added to AdobeDRMContainer.h file